lua/kubectl/actions/kube.lua

```diff
@@ -40,7 +40,7 @@ function M.start_kubectl_proxy(callback)
   local handle = vim.system(command.args, {
     clear_env = true,
     env = command.env,
-    stdin = false,
+    stdin = nil,
```

Comment:
It should be false according to the docs, not sure why lsp is saying something else?

![image](https://github.com/user-attachments/assets/eeffd457-859e-45b9-a72c-b067ad807b3c)

Reply:
My interpretation was that it's evaluated to false if nil. Maybe it's clearer to simply not pass `stdin` when not using it?

lua reference:
`--- @field stdin? string|string[]|true`

Reply:
better be explicit than implicit no? @paulbrittain

Reply:
Fair enough, then it's a Lua typing issue that causes this ugly assignment mismatch diagnostic because it's not typed `|boolean`. I'll just revert to false  :)

![image](https://github.com/user-attachments/assets/eb5885bf-76c7-4384-994a-cc5c79847283)

Reply:
Yeah your interpretation was probably correct @paulbrittain , I must confess that it's a weird thing to support only type true and not allow false when the default value later on is false....if you still have this error, I don't mind using nil here since it's basically the same as false.

Reply:
can't we do a comment casting? something like `---@as boolean`?

Reply:
I'm fine with both ways! we should aim not to have any warnings if possible, they can easily get out of hand and then no one knows what's important or a false alarm

Reply:
I agree on passing nil, that should also satisfy @mosheavni point on being explicit!